.gitignore

```diff
@@ -30,5 +30,22 @@ testfile
 .neofs-cli.yml
 
 # debhelpers
-**/.debhelper
+**/*debhelper*
```

Comment:
`-dirty` suffix is appended only if files in the working tree (committed or staged) are altered. None of these patterns is in tree. Why does it solve the problem?

Reply:
deb package, when building, download a dependencies and create some files in worktree, after build package i'm see next: 
``
git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   debian/changelog

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	debian/debhelper-build-stamp
	debian/files
	debian/frostfs-adm.bash-completion
	debian/frostfs-cli.bash-completion
	debian/frostfs-cli.debhelper.log
	debian/frostfs-cli.substvars
	debian/frostfs-cli/
	debian/frostfs-ir.debhelper.log
	debian/frostfs-ir.substvars
	debian/frostfs-ir/
	debian/frostfs-storage.debhelper.log
	debian/frostfs-storage.postrm.debhelper
	debian/frostfs-storage.substvars
	debian/frostfs-storage/
	man/
``
this files are make version dirty, because they always created on build package.

On this mask created:
``
debian/debhelper-build-stamp
debian/frostfs-cli.debhelper.log
debian/frostfs-ir.debhelper.log
debian/frostfs-storage.debhelper.log
debian/frostfs-storage.postrm.debhelper
``

on you gitignore was a ``**/.debhelper`` but this not working ....